contrib/sssd.spec.in

```diff
@@ -189,8 +198,13 @@ Requires: (sssd-nfs-idmap = %{version}-%{release} if libnfsidmap)
 Requires: libsss_idmap = %{version}-%{release}
 Requires: libsss_certmap = %{version}-%{release}
 %if 0%{?rhel}
+%if %{use_sysusers}
+BuildRequires:  systemd-rpm-macros
```

Comment:
AFAIK BuildRequires should go to the top where other build dependencies are specified.

Reply:
You also have to spaces before the package name.

Reply:
Fixed.